chaincfg/params.go

```diff
@@ -397,11 +408,11 @@ var MainNetParams = Params{
 
 	// BIP44 coin type used in the hierarchical deterministic path for
 	// address generation.
-	HDCoinType: 2,
```

Comment:
we likely don't want to change this as users may have generated addresses witht this path. however this means we should also update bip32 prefixes to ltub etc.